src/client/datascience/notebook/vscodeNotebookController.ts

```diff
@@ -379,12 +379,16 @@ export class VSCodeNotebookController implements Disposable {
         // Before we start the notebook, make sure the metadata is set to this new kernel.
         await updateNotebookDocumentMetadata(document, this.documentManager, selectedKernelConnectionMetadata);
 
+        if (document.notebookType === InteractiveWindowView) {
+            // Possible its an interactive window, in that case we'll create the kernel manually.
+            return;
```

Comment:
Not sure I follow this part here?

Reply:
We don't have the rsource information to create a kernel.
With interactive window, we have a *.py file, that needs to be passed when creating a kernel, else its not associated with any py file & thats not correct.

Reply:
I don't think we should skip initialization here, I'm already working on a code change to not cache the kernel in the nativeInteractiveWindow.

Reply:
> I don't think we should skip initialization her

Unless you have a way to pass the path of the Python file, then please feel free to restore this change. Else we cannot have this, as that will end up with the wrong information in the kernel.
I'll submit a subsequent PR that will make this resource information mandatory, that will make the problem i'm highlighting more obvious & we won't create kernels with the wrong information.